server/service/osquery.go

```diff
@@ -877,6 +887,20 @@ func (svc *Service) SubmitDistributedQueryResults(
 
 		detailUpdated = detailUpdated || ingestedDetailUpdated
 		additionalUpdated = additionalUpdated || ingestedAdditionalUpdated
+
+		if stats, ok := stats[query]; ok {
+			queryTime := stats.UserTimeMs + stats.SystemTimeMs
+			if queryTime >= fleet.QueryExcessiveCPUUsageThreshMs {
+				level.Warn(svc.logger).Log(
+					"msg", "distributed query CPU usage is excessive",
+					"hostID", host.ID,
+					"platform", host.Platform,
+					"query", query,
+					"queryTimeMs", queryTime,
```

Comment:
Since this is a debug statement, what do you think abut reporting `stats.UserTimeMs` and `stats.SystemTimeMs` instead of just `queryTime`?